milestone3.md

```diff
@@ -0,0 +1,23 @@
+# Milestone 3
+
+## Prototype Screenshots
+
+The home screen has a search bar where users can input the location they are looking for housing at. This leads to the map page where users can view listings and adjust sorting preferences. The user also has 
```

Comment:
I'm not sure if this was intended but the last sentence seems to be incomplete `The user also has`